core/models/start.go

```diff
@@ -175,6 +175,11 @@ func (s *FlowStart) WithParams(params []byte) *FlowStart {
 	return s
 }
 
+func (s *FlowStart) WithCreatedByID(userID UserID) *FlowStart {
```

Comment:
not needed - the only place we create a flow start with a createdBy, we don't use these methods